src/Pos/Launcher/Runner.hs

```diff
@@ -422,7 +422,9 @@ bracketDHTInstance BaseParams {..} action = bracket acquire release action
     instConfig =
         KademliaDHTInstanceConfig
         { kdcKey = bpDHTKey
-        , kdcHost = maybe "0.0.0.0" fst bpNetworkAddress
+        --, kdcHost = maybe "0.0.0.0" fst bpNetworkAddress
+        , kdcHost = "0.0.0.0" -- Amazon servers have different ifconfig (local) ip and public ip
+                              -- so we can't bind on public directly.
```

Comment:
But then again, the system assumes that if you use Kademlia, then the address of the UDP socket that Kademlia uses is the same as that of the network-transport-tcp public address. We should build this in and make it impossible to violate.